roles/openshift_cluster_monitoring_operator/README.md

```diff
@@ -0,0 +1,29 @@
+# OpenShift Cluster Monitoring Operator
+
+The OpenShift Cluster Monitoring Operator role manages the TPO deployment.
```

Comment:
If this is the first time this acronym is used it should fully state "Tectonic Prometheus Operator (TPO)"

Reply:
I thought we wanted to avoid naming anything "Tectonic" when it's part of OpenShift. I agree this line is not ideal, I need to think of some better phrasing